source/fem-dem/void_fraction.cc

```diff
@@ -0,0 +1,1141 @@
+#include <core/lethe_grid_tools.h>
```

Comment:
Should the license header be here as well?

Reply:
Yes it should